src/Core/Payment/PaymentOptionFormDecorator.php

```diff
@@ -35,8 +35,8 @@ class PaymentOptionFormDecorator
     public function addHiddenSubmitButton($formHTML, $optionId)
     {
         $doc = new DOMDocument();
-
-        $doc->loadHTML($formHTML);
+        
+        $doc->loadHTML('<meta http-equiv="Content-Type" content="text/html; charset=utf-8">' . $formHTML);
```

Comment:
I don't think it's a good idea to add a meta tag everywhere we ask for a hidden button...
It should be added to the header